app/Http/Controllers/InstructorsController.php

```diff
@@ -71,7 +71,21 @@ public function edit(Instructor $instructor)
      */
     public function update(Request $request, Instructor $instructor)
     {
-        //
+        $validated = $request->validate([
+            'First' => 'required',
+            'Last' => 'required',
+            'Display' => 'required',
+            'Email' => 'required',
+            'Active' => ''
+        ]);
+        $instructor->First = $validated['First'];
+        $instructor->Last = $validated['Last'];
+        $instructor->Display = $validated['Display'];
+        $instructor->Email = $validated['Email'];
+        $instructor->Active = $validated['Active'];
+        $request->session()->flash('alert-success', 'Saved instructor successfully!');
+        $instructors = Instructor::all();
+        return view('instructors.index', compact('instructor', 'instructors'));
```

Comment:
You should instead do something like `$result = $instructor->update($request->all());` and then redirect back to the edit page, or the index page. Post routes that are performing database actions should always redirect back to view actions

Reply:
I made these changes. But I don't know how to do anything else.